packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanelMain.tsx

```diff
@@ -573,6 +625,14 @@ export function TransferPanelMain({
 
   useUpdateUSDCTokenData()
 
+  type NetworkListboxProps = {
+    disabled?: boolean
+    label: string
+    options: Chain[]
+    value: Chain
+    onChange: (value: Chain) => void
+  }
```

Comment:
this can be deleted because really we only need `onChange` for both `from` and `to` now